lib/blacklist.js

```diff
@@ -1,22 +1,52 @@
 const db = require('./db')
 
-function blocked (address, cryptoCode) {
+// Get all blacklist rows from the DB "blacklist" table
+const getBlacklist = () => {
+  return db.any('select * from blacklist').then(res =>
+    res.map(item => ({
+      cryptoCode: item.crypto_code,
+      address: item.address,
+      createdByOperator: item.created_by_operator
+    }))
+  )
+}
+
+// Delete row from blacklist table by crypto code and address
+const deleteFromBlacklist = (cryptoCode, address) => {
+  return db.none(
+    'delete from blacklist where crypto_code = $1 and address = $2;',
+    [cryptoCode, address]
+  )
+}
+
+const insertIntoBlacklist = (cryptoCode, address) => {
+  return db
+    .any(
+      'insert into blacklist(crypto_code, address, created_by_operator) values($1, $2, $3);',
+      [cryptoCode, address, true]
+    )
+    .then(() => {
+      return { cryptoCode, address }
+    })
+}
+
+function blocked(address, cryptoCode) {
```

Comment:
Hmm, this doesn't pass standard. Functions should have a space between the name and parenthesis on function definition.